docs/fundamentals/integrations-overview.md

```diff
@@ -14,168 +14,113 @@ ms.topic: conceptual
+| [Redis](../caching/stackexchange-redis-integration.md) <br/> <img src="media/icons/redis-cube-red_white-rgb.png" alt="Redis logo." role="presentation" width="64" data-linktype="relative-path"> | - **Hosting**: [Aspire.Hosting.Redis](https://www.nuget.org/packages/Aspire.Hosting.Redis), [Aspire.Hosting.Garnet](https://www.nuget.org/packages/Aspire.Hosting.Garnet), or [Aspire.Hosting.Valkey](https://www.nuget.org/packages/Aspire.Hosting.RValkeyedis)<br>- **Client**: [Aspire.StackExchange.Redis](https://www.nuget.org/packages/Aspire.StackExchange.Redis) | A library for accessing [Redis](https://stackexchange.github.io/StackExchange.Redis/) caches. |
+| [Seq](../logging/seq-integration.md) <br/> <img src="media/icons/Seq_logo.256x.png" alt="Seq logo." role="presentation" width="64" data-linktype="relative-path"> | - **Hosting**: [Aspire.Hosting.Seq](https://www.nuget.org/packages/Aspire.Hosting.Seq)<br>- **Client**: [Aspire.Seq](https://www.nuget.org/packages/Aspire.Seq) | A library for logging to [Seq](https://datalust.co/seq). |
+| [SQL Server Entity Framework Core](../database/sql-server-entity-framework-integration.md) <br/> <img src="media/icons/SQL_256x.png" alt="SQL logo." role="presentation" width="64" data-linktype="relative-path"> | - **Hosting**: [Aspire.Hosting.SqlServer](https://www.nuget.org/packages/Aspire.Hosting.SqlServer)<br>- **Client**: [Aspire.Microsoft.EntityFrameworkCore.SqlServer](https://www.nuget.org/packages/Aspire.Microsoft.EntityFrameworkCore.SqlServer) | A library for accessing [SQL Server databases using Entity Framework Core](/ef/core/providers/sql-server/). |
+| [SQL Server](../database/sql-server-integration.md) <br/> <img src="media/icons/SQL_256x.png" alt="SQL logo." role="presentation" width="64" data-linktype="relative-path"> | - **Hosting**: [Aspire.Hosting.SqlServer](https://www.nuget.org/packages/Aspire.Hosting.SqlServer)<br>- **Client**: [Aspire.Microsoft.Data.SqlClient](https://www.nuget.org/packages/Aspire.Microsoft.Data.SqlClient) | A library for accessing [SQL Server](/sql/sql-server/) databases. |
+<!-- markdownlint-enable MD033 MD045 -->
 
-For more information, see [.NET dependency injection](/dotnet/core/extensions/dependency-injection).
+For more information on working with .NET Aspire integrations in Visual Studio, see [Visual Studio tooling](setup-tooling.md#visual-studio-tooling).
 
-### Keyed services
+**Azure specific resources are available in the following NuGet packages:**
+
+<span id="azure-hosting-libraries"></span>
```

Comment:
I would prefer if we had the same layout as above.

Reply:
For the Azure specific bits, do we want those in their own table, or interleaved with the table above?

Reply:
Own table.